servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -280,7 +286,7 @@ public synchronized JournalEntry toJournalEntry() {
     return new InodeFileEntry(getCreationTimeMs(), getId(), getName(), getParentId(), isPersisted(),
         isPinned(), getLastModificationTimeMs(), getBlockSizeBytes(), getLength(), isCompleted(),
         isCacheable(), mBlocks, mTTL,
-        new PermissionStatus(getUsername(), getGroupname(),getPermission()));
+        getUsername(), getGroupname(),getPermission());
```

Comment:
@shenguoquan, in case you didn't know, auto-formatting will be handy. In you're using Eclipse, press cmd+shift+F on the code block that you want formatting.